config_test.go

```diff
@@ -22,3 +23,25 @@ func TestConfigValidate(t *testing.T) {
 		t.Fatal(err)
 	}
 }
+
+// TestModuleCall_DependsOn is explicitly written to test the `depends_on`
+// within a module call. This was created manually as a way of validating
+// parsing of `depends_on` without needing to update test tooling to support
+// terraform 0.13.
+func TestModuleCall_DependsOn(t *testing.T) {
```

Comment:
Honestly without the actual output from a tf plan this seems a bit useless. I'm +0 on this test, so if it's satisfying for you please merge it.

One thing though is that it might be good to point a bright orange flag around it so we know to remove it when we get automated tests for 0.13 building. Maybe put it in its own file?

Reply:
Yeah, it's really just a silly test.